libs/ledger-live-common/src/exchange/swap/getExchangeRates.ts

```diff
@@ -146,11 +147,29 @@ const inferError = (
     maxAmountFrom: string;
     errorCode?: number;
     errorMessage?: string;
+    status?: string;
   }
 ): Error | CustomMinOrMaxError | undefined => {
   const tenPowMagnitude = new BigNumber(10).pow(unitFrom.magnitude);
-  const { amountTo, minAmountFrom, maxAmountFrom, errorCode, errorMessage } =
-    responseData;
+  const {
+    amountTo,
+    minAmountFrom,
+    maxAmountFrom,
+    errorCode,
+    errorMessage,
+    status,
+  } = responseData;
+
+  // DEX quotes are out of limit scope error
```

Comment:
> DEX quotes are out of limit scope error

This does not make sense to me.
What do you mean by _out of limit scope_?
IMO it's either _out of scope_ or _out of limit_ (and in both case, what is the _limit_ or _scope_?)

Reply:
DEX quotes are out of limits error. We do not know if it is a low or high limit, neither the amount.